sites/default/sqlconf.php

```diff
@@ -1,30 +1,31 @@
 <?php
```

Comment:
This needs to be reverted. There are several strategies that folks use to prevent this. Some things to consider:
1. Don't run openemr from within the git repo
2. If do run openemr within the git repo, then use the following cool docker method that does not mess up your git repo:
https://github.com/openemr/openemr/blob/master/CONTRIBUTING.md#code-contributions-local-development